### PR TITLE
TC : Search and Add product

### PR DESCRIPTION
**Pull request by NFartenadze**

### CODE REVIEW

**Comment by NFartenadze** [comment]
I committed TC: View Product Details here.

**Comment by NFartenadze** [comment]
Committed   TC:Verify Navigation Links and Navigation Functionality here to not lose time

File: cypress/e2e/duckduckgo.feature
Comment (FCosta20) [inline cypress/e2e/duckduckgo.feature:1]: Why are you creating this feature and test?

File: cypress.config.ts [hunk 1]
```diff
@@ -1,12 +1,55 @@
...
 export default defineConfig({
   e2e: {
-    setupNodeEvents(on, config) {
-      // implement node event listeners here
-    },
-    // baseUrl: "https://www.gsmarena.com/",
-    viewportHeight: 800,
-    viewportWidth: 1440,
+    baseUrl: "https://duckduckgo.com",
+    specPattern: "**/*.feature",
```
Comment (FCosta20) [inline cypress.config.ts:52]: You are configuring only .feature files but you also have tests that don't use cucumber, please choose if you want to use cucumber or not

**Review by FCosta20 — CHANGES_REQUESTED** [review]

File: cypress/e2e/duckduckgo.feature
Comment (NFartenadze) [reply cypress/e2e/duckduckgo.feature:1]: i found how to setup project for cucumber, and i just copied the config with example test

File: cypress.config.ts (hunk 1 above)
Comment (NFartenadze) [reply cypress.config.ts:52]: i started looking into cucumber just at the end of day, that's why, i will migrate existing tests into cucumber tests

File: cypress/e2e/duckduckgo.feature
Comment (FCosta20) [reply cypress/e2e/duckduckgo.feature:1]: But this test is not needed, don't push it

File: cypress.config.ts (hunk 1 above)
Comment (FCosta20) [reply cypress.config.ts:52]: Ok, do that and then assign it to me for review

**Pull request merged**